app/controllers/keywords_controller.rb

```diff
@@ -3,10 +3,17 @@
 require 'csv'
 
 class KeywordsController < ApplicationController
-  def index; end
+  include Pagy::Backend
+
+  def index
+    @pagy, keywords = pagy(current_user.keywords)
+    @decorated_keywords = keywords.map(&:decorated)
+  end
 
   def create
-    parse_keywords
+    parse_keywords.each do |key|
```

Comment:
How about to use full name?
```suggestion
    parse_keywords.each do |keyword|
```

Reply:
Fixed in [`f42675d` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/f42675dd1e9b7fdfe0b06e96e5e8709d750dd695) 🙈